code/modules/detectivework/evidence.dm

```diff
@@ -47,6 +47,7 @@
 	I.pixel_x = 0		//then remove it so it'll stay within the evidence bag
 	I.pixel_y = 0
 	var/image/img = image("icon"=I, "layer"=FLOAT_LAYER)	//take a snapshot. (necessary to stop the underlays appearing under our inventory-HUD slots ~Carn
+	img.plane = HUD_PLANE
```

Comment:
You have the byond reference at your fingertips and yet you choose to not use it and instead doubt